docs/post-response-codes.md

```diff
@@ -0,0 +1,121 @@
+# PostResponseCodes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Async-V1-11
+
+## Output Message
+
+Synchronous POST operations must have responses with either 200, default OR 204, default return codes. They also must not have other response codes.
+Long-running POST operations must have responses with 202, default return codes and should also have a 200 return code only if the final response is intended to have a schema, if not the 200 return code must not be specified. They also must not have other response codes.
```

Comment:
Synchronous POST operations must have one of the following combinations of responses - 200 and default ; 204 and default. They also must not have other response codes.